### DISCUSSION TITLE
all symbols are expanded without hierarchy structure

### DISCUSSION THREAD

**Discussion by lainegates** [hedyhli/outline.nvim#89] [Help] [answered]
Hi, outline.nvim is a very great nvim plugin.  I use it everyday.
Today I press somekey, then all symbols are expanded with hierarchy structure. Does anyone know how to reset it ?
I tried to uninstall it and reinstall, which doesnot work.
![image](https://github.com/user-attachments/assets/ab759321-bd11-42f8-b10e-3c5115915acc)

**Comment by hedyhli** [comment]
Yes, unfortunately the LSP server for Python you're using does not support symbol hierarchies. I experience this issue as well and there's not much we can do on outline.nvim's side to fix this.

You can only either find an LSP with hierarchies support or use tree-sitter once outline.nvim supports it (#13).

**Reply by hedyhli** [reply, marked as answer]
If you previously have the hierarchies with Python files, then perhaps you're using another LSP at the time.
